test/unittests/tool_commands_test.cpp

```diff
@@ -36,6 +36,16 @@ TEST(tool_commands, run_empty_code)
     EXPECT_EQ(out.str(), out_pattern("Frontier", 1, "success", 0, ""));
 }
 
+TEST(tool_commands, run_oog)
```

Comment:
Is this really part of the "allow optional 0x prefix" commit?

Reply:
Yes, because I use 0x in this test :) I also triggered some previously uncovered lines BTW.